tests/smoke/task.bats

```diff
@@ -12,8 +12,8 @@
     l0 task create --wait test task1 alpine:latest
 }
 
-@test "task create --copies 2 --env alpine:key=val test task2 alpine:latest" {
-    l0 task create --copies 2 --env alpine:key=val test task2 alpine:latest
+@test "task create --env alpine:key=val test task2 alpine:latest" {
```

Comment:
OK, yes, so this would be a breaking change. Not the end of the world, but we need to have a good explanation as for why we wouldn't want `copies` as a param any more.

Reply:
+1

Reply:
Doesn't "breaking change" imply we would need to bump to v0.11? If so, that is not true; these changes are still compatible with v0.10 APIs.

Reply:
it's a breaking change insofar as Chunnels re-implemented their workflows to use `copies`. Also, it changes the params for task create which in turn changes the request model, doesn't it?

Reply:
Also, I'm still not entirely clear on how removing `copies` reduces the amount of API calls.

Reply:
It does change the request model but not in an incompatible way. Using an older CLI to create tasks against this would just ignore the copies param.

Reply:
However, to make the chunnels lives easier, we could actually keep the `copies` param in the cli. under the covers, it would just make a bunch of calls to the api.